__tests__/Port.test.js

```diff
@@ -0,0 +1,39 @@
+const Port = require("../src/Port");
+
+describe("Port", () => {
+  describe("ports and ships", () => {
+    let port;
+    let ship;
+    let titanic;
+    let queenMary;
+
+    beforeEach(() => {
+      port = new Port("name");
+      ship = jest.fn();
+      titanic = jest.fn();
+      queenMary = jest.fn();
+    });
+
+    it("can be instantiated", () => {
+      expect(new Port()).toBeInstanceOf(Object);
+    });
+
+    it("port has a name", () => {
+      expect(port.name).toBeDefined();
+    });
+
+    it("can add a ship", () => {
+      port.addShip(ship);
+
+      expect(port.ships).toContain(ship);
+    });
+
+    it("can remove a ship", () => {
+      port.addShip(titanic);
+      port.addShip(queenMary);
+      port.removeShip(queenMary);
+
+      expect(port.ships).toEqual([titanic]);
+    });
```

Comment:
This test passes despite your broken `removeShip` functionality because you remove the last ship you added. If you were you to change your test to:
```
port.addShip(titanic);
port.addShip(queenMary);
port.removeShip(titanic);

expect(port.ships).toEqual([queensMary]);
```
you'll see that the test fails

Reply:
My code is what the track walkthrough I suggested i write. I have amended the functionality, do I still need to amend this test?